cfme/fixtures/appliance.py

```diff
@@ -87,25 +87,28 @@ def temp_appliance_preconfig(temp_appliance_preconfig_modscope):
 
 
 @pytest.fixture(scope="module")
-def temp_appliance_preconfig_modscope(appliance, pytestconfig):
+def temp_appliance_preconfig_modscope(request, appliance, pytestconfig):
     with sprout_appliances(appliance, config=pytestconfig, preconfigured=True) as appliances:
```

Comment:
I just want to note I was considering to put the `_collect_logs` into the `sprout_appliances`. I do not remember why I didn't do it. Perhaps because I would have to pass the `request` there.